app/assets/javascripts/application.js

```diff
@@ -22,4 +22,8 @@ $(document).ready(function(){
 // parralax
 $(document).ready(function(){
   $('.parallax').parallax();
-});
\ No newline at end of file
+});
+
+$(document).ready(function() {
```

Comment:
Mixed double and single quotes.<br>Identifier 'material_select' is not in camel case.